olive/workflows/run/config.py

```diff
@@ -125,7 +124,6 @@ def validate_evaluation_only(cls, v):
 
     @validator("passes", pre=True, each_item=True)
     def validate_pass_host_evaluator(cls, v, values):
-        v = _resolve_system(v, values, "host")
```

Comment:
I don't think this is deadcode, we use the `pass_config.host` here https://github.com/microsoft/Olive/blob/main/olive/workflows/run/run.py#L153. Users are able to specify pass specific hosts if they want.

Reply:
I could not find the the host for pass is configured in either olive_pass.py or in pass_config.py. where is the schema defined?